packages/sdk-middleware-http/test/http.spec.js

```diff
@@ -239,6 +238,42 @@ describe('Http', () => {
     }),
   )
 
+  it('handle failed response (network error)', () =>
```

Comment:
i would add more tests:
- that repeater is not executed on 400/409/or other not 5xx errors
- that it rejects after X failed retries with last/most recent error response
- that it resolves after X successful retry